pkg/controller/node/nodecontroller.go

```diff
@@ -123,7 +126,9 @@ func NewNodeController(
 		kubeClient:             kubeClient,
 		recorder:               recorder,
 		podEvictionTimeout:     podEvictionTimeout,
-		podEvictor:             podEvictor,
+		maximumGracePeriod:     5 * time.Minute,
+		podEvictor:             NewRateLimitedTimedQueue(podEvictionLimiter, false),
+		terminationEvictor:     NewRateLimitedTimedQueue(podEvictionLimiter, false),
```

Comment:
Do we really want to share the same RateLimiter between podeEvictor and terminationEvictor? I'm not against it, but in general I don't like sharing state if it gives nothing except coding convenience.